Gandalf/build.sh

```diff
@@ -124,21 +100,18 @@ echo "Bundling ${GANDALF_COMMAND_NAME}..."
 
 cp "gandalf" "${PKG_PACKAGE}/usr/bin"
 
-#Make it executable
+#Make it executable 
 chmod +x "${PKG_PACKAGE}/usr/bin/gandalf"
 
 #Copy the DEBIAN scripts
 echo "Copying the DEBIAN scripts"
 
 cp "prerm" "${PKG_PACKAGE}/DEBIAN"
 cp "postinst" "${PKG_PACKAGE}/DEBIAN"
-echo
-read -p  "${bold}${red}--- If you have anything else you need to put into Gandalf, now's the time. If you're finished or you don't want to add something press any key to continue. ---${normal}"
-echo
 
```

Comment:
Let it here: at the top it isn't needed (my opinion)
And the question is: why do we need this?